### ISSUE TITLE
Does the repository include all necessary configuration files?

### ISSUE THREAD

**Issue by kkytola** [alma-n/Polya-lean#2]
I did not manage to make Lean work in the forked/cloned repository. I think some configuration files may be missing or may have something wrong in them

Did anyone else succesfully clone and have it working? We can debug in a meeting.

**Comment by kkytola** [comment]
I tried with another computer. I now think the only thing that does not work is `import Polya.MiscLemmas` in my pull-request (but I get the same failure for any other import within the Polya repository). The error message is:
`unknown package 'Polya'
You might need to open '...' as a workspace in your editor`

But I opened the folder exactly the same way I always do. So I still suspect there is something wrong in the configurations.
